packages/web-components/@sfx/autocomplete/test/unit/autocomplete.test.ts

```diff
@@ -88,5 +98,50 @@ describe('Autcomplete Component', () => {
 
       expect(autocomplete.results).to.deep.equal(results);
     });
+
+    it('should set the results property to an empty array if the response data is undefined', () => {
```

Comment:
@geoffcoutts Can you elaborate on this? Do you mean if the event payload contains an error because the autocomplete call failed? This test ensures that the products are set to an array even if there are no results, which should handle any kind of response that does not include results. Let me know.

Reply:
@DanielMoniz-GBI  True, it would set it to blank regardless. I'm guessing any errors that would potentially be passed to this function from search calls would also show up as either a failed call in the network tab or in the search call's response?

Reply:
Yeah, I think any errors would have to be dealt with in the logic layer. If it's making it this far, we're assuming it's arriving in a certain format.

We also have a separate event for error responses, I believe, so we can assume that this event contains the results from a successful search.